### ISSUE TITLE
Enhancement:

### ISSUE THREAD

**Issue by SilasK** [BigDataBiology/SemiBin#44]

**Comment by luispedro** [comment]
This is a good point. We did pick one specific version as default, but in principle, any version of GTDB would work well for SemiBin (@psj1997, right?).

Maybe this needs to be clearer in the docs.

**Comment by psj1997** [comment]
Yes. Actually any version is fine.

**Comment by SilasK** [comment]
If it would be possible to use the same GTDB data as I downloaded for GTDB-tk I would save 50GB of storage space.

**Comment by luispedro** [comment]
@SilasK : What have you tried?

If you point SemiBin to the downloaded data (using the `-r` parameter), it should work.

**Comment by SilasK** [comment]
Sorry I didn't try anything. The database folder for SemiBin consists of a 50gb mmseqs database, where the GTDB-tk It contains many other files including the fasta of the reference. 

Is there a way I can specify a different GTDB version?

the -r parameter only specifies the folder where to download it, isn't it?

**Comment by SilasK** [comment]
By the way, did you try the simple mmseqs taxonomy annotation? It's probably less comprehensive than GTDB but you would also include Eukaryotes. For now, SemiBin is biased towards prokaryotes.

**Comment by psj1997** [comment]
> Sorry I didn't try anything. The database folder for SemiBin consists of a 50gb mmseqs database, where the GTDB-tk It contains many other files including the fasta of the reference.
> Is there a way I can specify a different GTDB version?
> the -r parameter only specifies the folder where to download it, isn't it?

Yes, you can use any version that mmseqs2 used and specify the folder with -r parameter.

> By the way, did you try the simple mmseqs taxonomy annotation? It's probably less comprehensive than GTDB but you would also include Eukaryotes. For now, SemiBin is biased towards prokaryotes.

We do not, but it seems a good idea to try. What do you mean of the simple mmseqs taxonomy annotation?

Sincerely
Shaojun

**Comment by SilasK** [comment]
I meant this: https://github.com/soedinglab/mmseqs2#taxonomy
